targets/bcm27xx-bcm2711

```diff
@@ -0,0 +1,7 @@
+include 'bcm27xx.inc'
+
+device('raspberrypi-4-model-b', 'rpi-4', {
+	manifest_aliases = {
+		'raspberry-pi-4-model-b-rev-1.1',
+	}
```

Comment:
```suggestion
	},
```
I would expect a trailing comma here... can we somehow configure our linter to complain in such cases?